package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "sheltertech.org",
+  "version": "1.0.0",
+  "description": "Sheltertech Info Site",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "gatsby develop",
+    "build": "gatsby build"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ShelterTechSF/sheltertech.org.git"
+  },
+  "author": "Sheltertech.org",
+  "license": "MIT",
```

Comment:
We actually use the GPLv3 license for all ShelterTech open source projects. See https://github.com/ShelterTechSF/askdarcel-web/blob/aab487c081e87ba6804fefea05f3ed4f0908c71e/package.json#L101 for what the line looks like in the package.json from the askdarcel-web project.